### DISCUSSION TITLE
Sansuang Odyssey Ark 55  is not working

### DISCUSSION THREAD

**Discussion by JunwayGuo** [MonitorControl/MonitorControl#1518] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

For Sansuang Odyssey Ark 55 the monitor is not working and the volume cannot be adjusted.
Connected via HDMI 2.1.

### Steps to reproduce

...

### Expected behavior

I can adjust the sound, but you can't adjust the volume

### Environment Information (please complete the following information)

```markdown
- macOS version: 14.3
- Mac model: MacPro 2019
- MonitorControl version: 4.2
- Monitor(s): Sansuang Odyssey Ark 55
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
Hi there, many Samsung displays, especially those that are actually TVs internally do not support DDC. For volume control you can try software audio filter based solutions like eqMac.
